### PR TITLE
Add deployment instructions for PowerStore

### PR DESCRIPTION
**Pull request by ivanpch**

### CODE REVIEW

**Review by vlad-belogrudov — APPROVED** [review]

**Review by gaelrehault — APPROVED** [review]

File: osp-deploy/cinder/README.md [hunk 1]
```diff
@@ -200,6 +201,50 @@ parameter_defaults:
...
+
+With a director deployment, PowerStore backend can be deployed using the integrated heat environment file. This file is located in the following path of the Undercloud node:
+/usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-powerstore-config.yaml
+
+Copy this file to a local path where you can edit and invoke it later. For example, to copy it to ~/templates/:
+
+```bash
+$ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-powerstore-config.yaml ~/templates/
+```
+Afterwards, open the copy (~/templates/cinder-dellemc-powerstore-config.yaml) and edit it as you see fit. The following shows a sample content of the file. The files will list optional params that the user can choose to override if they don't like the default value.
```
Comment (arkadykanevsky) [inline osp-deploy/cinder/README.md:217]: override -> overwrite

File: osp-deploy/cinder/README.md [hunk 2]
```diff
@@ -200,6 +201,50 @@ parameter_defaults:
...
+
+```yaml
+# A Heat environment file which can be used to enable a
+# Cinder Dell EMC PowerMax backend, configured via puppet.
```
Comment (arkadykanevsky) [inline osp-deploy/cinder/README.md:221]: PowerMax -> Powerstore

File: osp-deploy/cinder/README.md [hunk 3]
```diff
@@ -200,6 +201,50 @@ parameter_defaults:
...
+  CinderPowerStoreMultiConfig: {}
+  CinderPowerStoreAvailabilityZone: ''
+  CinderPowerStoreSanIp: '10.10.10.10'
+  CinderPowerStoreSanLogin: 'admin'
+  CinderPowerStoreSanPassword: 'password'
+  CinderPowerStoreAppliances: 'appliance-1,appliance2'
+  CinderPowerStorePorts: '10.10.10.11,10.10.10.12'
+  CinderPowerStoreStorageProtocol: 'iSCSI'
+
+# To configure multiple PowerStore backends, use CinderPowerStoreMultiConfig to
```
Comment (arkadykanevsky) [inline osp-deploy/cinder/README.md:237]: example above already covers two powerstores.

#To configure multiple PowerStore backends, use CinderPowerStoreMultiConfig to
# assign parameter values specific to that backend. For example:
->
#To configure multiple PowerStore backends differently, use CinderPowerStoreMultiConfig to
# assign parameter values specific to each backend. For example:

**Review by arkadykanevsky — CHANGES_REQUESTED** [review]
small editorial changes needed

File: osp-deploy/cinder/README.md (hunk 3 above)
Comment (arkadykanevsky) [reply osp-deploy/cinder/README.md:237]: Ivan, can you make these small changes so we can merge the patch?

**Review by arkadykanevsky — APPROVED** [review]

**Pull request merged**